### ISSUE TITLE
No Stream Avaliable

### ISSUE THREAD

**Issue by DJSleeves** [Tempest0580/tempest#101]
I'm trying to use tempest on kodi and every show and movie I choose, it states "no stream available".  I was just wondering if anyone else was having this problem and if there is any way to fix it. 

I am a HUGE fan of this add-on. Thank you.

**Comment by honeybunn35** [comment]
Having the same issue.

**Comment by eeden003** [comment]
Yes, this/you are CORRECT.  About 1.5 weeks ago the site WATCHEPISODESERIES went off-line. Now several addons with free links have 1 or 2 or nó free links  ! I use the addons every day and the most free links came from WATCHEPISODESERIES so logical that there are now no streams. Maybe the developer(s) can add other sources that dó work ?

**Comment by DJSleeves** [comment]
I'm thankful that you guys are having the same issue and its not my ISP. 😊..Hopefully it gets fixed.  I'm patient though.  😊
